src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -93,9 +96,13 @@ private void testGeneralQuestionOperations() throws Exception {
         testNewQuestionLink();
         testInputValidationForQuestion();
         testAddQuestionAction();
+        
+        testResponseRateBeforeEditingQuestion();
 
         testEditQuestionLink();
         testEditQuestionAction();
+        
+        testResponseRateAfterEditingQuestion();
```

Comment:
It might be better to not test it this way. Instead, will it be better to include the tests for response rate to be in <code>testEditQuestionAction</code>?